drivers/serial/uart_ns16550.h

```diff
@@ -11,6 +11,8 @@
 #ifndef ZEPHYR_DRIVERS_SERIAL_UART_NS16550_H_
 #define ZEPHYR_DRIVERS_SERIAL_UART_NS16550_H_
 
-#define CMD_SET_DLF	0x01
+#include <zephyr/drivers/serial/uart_ns16550.h>
+
+#warning "drivers/serial/uart_ns16550.h" is deprecated, include <zephyr/drivers/serial/uart_ns16550.h> instead
```

Comment:
Those directories are counted from different base

Reply:
New version might be less confusing.